src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { name, sex, slug } = person;
+
+  return (
+    <Link
+      to={`/people/${slug}`}
+      className={cn('has-text-link', {
+        'has-text-danger': sex === 'f',
```

Comment:
It would be cool not to use magic values and put such strings into constants and reuse them:
```
const FEMALE = 'f';
```
This will increase the readability of the code and save us from possible errors.
This applies to development in general.